Gemfile

```diff
@@ -54,6 +54,3 @@ group :development do
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
-
-# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
```

Comment:
Hmm... You know, you're right.

To be fair, I'm not sure that @AkivaGreen's problem was "installing this gem.'"  I just head him say "tzinfo" and I said, "Isn't that Windows-only? Chuck it!"